src/Uno.UI/UI/Xaml/Input/Internal/FocusProperties.cs

```diff
@@ -266,7 +266,7 @@ internal static bool IsFocusEngaged(DependencyObject dependencyObject)
 
 		internal static bool HasFocusedElement(DependencyObject element)
 		{
-			//if (element->IsActive())
+			if (element is UIElement { IsInLiveTree: true })
```

Comment:
Is there a manual test that we can validate this behavior for, or is it just logging errors?

Reply:
Not really testable, as it is just about the log, the code otherwise passes anyway, and there is no behavior change, as it just short-circuits later... The log is actually just Debug level as well